Lib/bdb.py

```diff
@@ -125,12 +187,14 @@ def dispatch_exception(self, frame, arg):
     # definition of stopping and breakpoints.
 
     def is_skipped_module(self, module_name):
+        "Return True if module_name matches any skip pattern."
```

Comment:
Triple quotes are only needed for multiple lines.  For single lines, single quotes are commonly used, in spite of what PEP257 says.  I only changed your additions where line length was an issue.